python/cudf/cudf/utils/gpu_utils.py

```diff
@@ -1,13 +1,13 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
 
-def validate_setup(check_dask=True):
+def validate_setup():
     import os
 
     # TODO: Remove the following check once we arrive at a solution for #4827
     # This is a temporary workaround to unblock internal testing
     # related issue: https://github.com/rapidsai/cudf/issues/4827
-    if not check_dask and "RAPIDS_NO_INITIALIZE" in os.environ:
+    if "RAPIDS_NO_INITIALIZE" in os.environ:
```

Comment:
We're doing both RAPIDS and library specific env vars :)
```suggestion
    if "RAPIDS_NO_INITIALIZE" in os.environ or "CUDF_NO_INITIALIZE" in os.environ:
```